partner_phonecall_schedule/models/res_partner.py

```diff
@@ -30,12 +32,12 @@ class ResPartner(models.Model):
     @api.depends("phonecall_calendar_ids", "phonecall_calendar_attendance_ids")
     def _compute_phonecall_available(self):
         """Know if a partner is available to call right now."""
-        Attendance = self.env["resource.calendar.attendance"]
+        attendance = self.env["resource.calendar.attendance"]
```

Comment:
This is upper on purpose, for knowing that this is the model handler. This is done the same on Odoo.